tests/integration/security/authorization_test.go

```diff
@@ -1290,6 +1296,7 @@ func TestAuthorization_Audit(t *testing.T) {
 						}
 						policies := tmpl.EvaluateAllOrFail(t, args, file.AsStringOrFail(t, tc.configFile))
 						t.Config().ApplyYAMLOrFail(t, ns.Name(), policies...)
+						util.WaitForConfig(t, policies[0], ns)
```

Comment:
should we switch WaitForConfig to be `WaitForConfig(t, ns, policies...)` so we can check all?

Probably doesn't matter, but could in some cases

Reply:
Good point! Let me update the PR later.

Reply:
Done.